grpc_opa/authorizer.go

```diff
@@ -343,3 +344,55 @@ func addObligations(ctx context.Context, opaResp OPAResponse) (context.Context,
 	}
 	return ctx, err
 }
+
+// NewAuthorizationConfigUnary returns new unary config for authorization requests
+func NewAuthorizationConfigUnary(application string, opts ...Option) *Config {
+	cfg := &Config{
+		address: opa_client.DefaultAddress,
+	}
+
+	for _, opt := range opts {
+		opt(cfg)
+	}
+
+	if cfg.authorizer == nil {
+		log.Info("authorizers empty, using default authorizer")
+		cfg.authorizer = []Authorizer{NewDefaultAuthorizer(application, opts...)}
+	}
+
+	return cfg
+}
+
+// AffirmAuthorizationUnary makes an authz request to sidecar-OPA
+// If authorization allowed, error returned is nil,
+// and a new context is returned containing any obligations
+// which caller must further evaluate if required
+func AffirmAuthorizationUnary(ctx context.Context, cfg *Config, fullMethod string, grpcReq interface{}) (context.Context, error) {
+	logger := ctxlogrus.Extract(ctx)
+	var (
+		ok     bool
+		newCtx context.Context
+		err    error
+	)
+
+	for _, auther := range cfg.authorizer {
```

Comment:
Don't you need the check for `cfg.authorizer == nil`  before here?

Reply:
```suggestion
	for _, authorizer := range cfg.authorizer {
```
or simply a or auth :)